lib/random_word.rb

```diff
@@ -69,8 +88,10 @@ def each()
     # Create a random, non-repeating enumerator for a list of words
     # (or anything really).
     def enumerator(word_list, list_of_regexs_or_strings_to_exclude = [])
+      @word_list = word_list
       word_list.extend EachRandomly
       word_list.random_word_exclude_list = list_of_regexs_or_strings_to_exclude
+      word_list.set_constraints(not_shorter_than: 1, not_longer_than: nil)
```

Comment:
Also I forgot that `set_constraints` will default the values itself if passed no args, so I can delete more characters.